db/migrate/20171115105030_add_reference_to_resource_monitor.rb

```diff
@@ -0,0 +1,5 @@
+class AddReferenceToResourceMonitor < ActiveRecord::Migration[5.0]
+  def change
+    add_column :resource_monitors, :reference, :string, null: false, index: true
```

Comment:
Do we know if there are any resource monitors in the database already which will cause a `null` constraint error when the column is created? (unfortunately integration is looking a bit unhealthy right now so I can't check)

Reply:
We created a couple of test monitors last week. So possibly if it has not been synced with staging/prod since then. Could throw in a `ResourceMonitor.all.destroy` ?

Reply:
I would add the column as nullable, fill it with nonsense references and then make the column not nullable - and same for `app` I guess.